examples/visualization/gradcam_demo.py

```diff
@@ -0,0 +1,83 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+
+from keras_cv import visualization
+from keras_cv.visualization.utils import normalize, collect_endpoints
+
+SIZES = (299, 299)
+LABELS = 5
+
+# Load images from the web
+def load_image(path):
+    image = tf.keras.preprocessing.image.load_img(path, target_size=SIZES)
+    return tf.keras.preprocessing.image.img_to_array(image)
```

Comment:
@lucasdavid The `tf.keras.preprocessing.image.*` will be depreciated, [source](https://github.com/keras-team/keras-cv/issues/320#issuecomment-1099518637)). But we can port that two function from `tf.keras.utils.*`.